### ISSUE TITLE
text overflows direction changes

### ISSUE THREAD

**Issue by ovidubya** [terrastruct/d2#494]
## Input
```d2
direction: right

Backend: {
    A
    Sample Request: |`json
        {
            "uuid": "abcd-efg"
        }
    `|
    B

}

Backend.A -> Backend.Sample Request: sends
Backend.Sample Request -> Backend.B: text will overflow to nodes
```

## Ouput
<img width="846" alt="Screen Shot 2022-12-21 at 4 16 50 PM" src="https://user-images.githubusercontent.com/22357156/209013067-b689393f-d877-4526-9718-b21bf6ab5ce0.png">

**Comment by gavin-ts** [comment]
should be fixed by #484

**Comment by alixander** [comment]
can confirm it is:

![d2 (53)](https://user-images.githubusercontent.com/3120367/209587068-61017f02-c7c6-4c41-8212-fcf55fe94824.png)

https://play.d2lang.com/?script=bJBBCsIwEEX3c4pP9vUAWQjNEfQCjc1UqjHRJrFC27tLAgYR_2LmD4_hM1PfJzHl64mU7q_sjMRCANCWetS3u2Uc-JE4RIm1uwTvCspaqssSKY1GSAh96k3Dw1lUvBXXraUpoq3m7Vo0e3yG37zAzgT6T7_3lETkV8Q8Wgv_5Gmwfkb0cN5woHcAAAD__w%3D%3D
